### PR TITLE
refactor!: options change

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: src/metrics/index.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export { Metrics } from './metrics';
-export { metricsMiddleware, defaultMetricsMiddleware, collectMetricsExpressMiddleware } from './middleware/metrics';
+// export { metricsMiddleware, defaultMetricsMiddleware, collectMetricsExpressMiddleware } from './middleware/metrics';
```
Comment (shimoncohen) [inline src/metrics/index.ts:2]: Remove this line?

Comment (CptSchnitz) [reply src/metrics/index.ts]: removed

File: package.json [hunk 2]
```diff
@@ -59,7 +64,8 @@
     "@opentelemetry/sdk-node": "^0.46.0",
     "@opentelemetry/sdk-trace-base": "^1.19.0",
     "@opentelemetry/sdk-trace-node": "^1.19.0",
-    "env-var": "^7.1.1",
+    "ajv": "^8.16.0",
+    "ajv-formats": "^3.0.1",
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply package.json:68]: no its not.
https://www.npmjs.com/package/ajv-formats/v/3.0.1
the package you linked is for different formats.

[the remark this answers is not in the document]
Comment (netanelC) [reply package.json:68]: Oopsi

**Review by netanelC — APPROVED** [review]

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**